src/schemas/compare.py

```
@@ -0,0 +1,34 @@
+# src/schemas/compare.py
+from pydantic import BaseModel
+from typing import List
```

Ruff (0.14.8)

`typing.List` is deprecated, use `list` instead

(UP035)